main/src/main/scala/sbt/internal/TaskProgress.scala

```diff
@@ -167,4 +167,16 @@ private[sbt] object TaskProgress {
   def cursorUp(n: Int): String = s"\u001B[${n}A"
   def cursorDown(n: Int): String = s"\u001B[${n}B"
   final val CursorDown1 = cursorDown(1)
+
+  def isEnabled: Boolean =
+    ConsoleAppender.formatEnabledInEnv && sys.props
+      .get("sbt.progress")
+      .flatMap({ s =>
+        ConsoleAppender.parseLogOption(s) match {
```

Comment:
Admittedly superficial, but `s` is used so frequently for state throughout sbt that I assumed this was an instance of `State` not `String`.

Reply:
I'll change to `str`.